v1/plugins/plugins.go

```diff
@@ -448,6 +448,10 @@ func New(raw []byte, id string, store storage.Store, opts ...func(*Manager)) (*M
 		f(m)
 	}
 
+	if m.parserOptions.RegoVersion == ast.RegoUndefined {
+		m.parserOptions.RegoVersion = ast.DefaultRegoVersion
+	}
```

Comment:
[question]: Is this where the "default to the `ast.DefaultRegoVersion` if none provided" logic needed to move to?

Reply:
Yes. If a `plugins.Manager` has `ast.RegoUndefined` as rego-version, then any consumer of the manager would simply do the correct thing depending on what v0/v1 package it comes from, which is why this has gone undetected. Defaulting to v1 here if no rego-version has been provided through the options is however more consistent with user expectations.